### DISCUSSION TITLE
Combobox/Dropdown Select All button

### DISCUSSION THREAD

**Discussion by erikrenz88** [Appsilon/shiny.fluent#211] [Q&A]
Just curious if I missed either of these inputs have functionality for a select/deselect all? I know Fluent UI supports but wanted to confirm its not support within shiny.fluent?

**Comment by jakubsob** [comment]
Hi @erikrenz88, the ComboBox Fluent component supports select all, although it's not supported out-of-the-box in shiny.fluent. Additional JavaScript handler would need to be added to the component to support this feature as in [ComboBox with select all option example](https://developer.microsoft.com/en-us/fluentui#/controls/web/combobox).

**Reply by erikrenz88** [reply]
Thanks @jakubsob - I will give it a shot. I saw there was some working going on with this feature, presumably to make it out of the box similar to DropDown Fluent component? #161?
